packages/inventory-compliance/src/ComplianceEmptyState.js

```diff
@@ -54,7 +88,14 @@ ComplianceEmptyState.defaultProps = {
         component="a"
         href="/insights/compliance/scappolicies">
         Create new policy
-    </Button>
+    </Button>,
+    client: new ApolloClient({
+        link: new HttpLink({
+            uri: COMPLIANCE_API_ROOT + '/graphql',
+            credentials: 'include'
+        }),
+        cache: new InMemoryCache()
+    })
```

Comment:
This might be surprising, because it looks like if you do:
```
import ComplianceEmptyState from ...;

<ApolloProvider client={client}>
  <ComplianceEmptyState/>
</ApolloProvider> 
```

it does not work. Apparently components that are imported as a dependency don't inherit the Apollo client from the context, so it has to have its own. In the case of the SystemDetails page, this would be unnecessary, but I kept this as the default so that the component is "batteries-included" and works without having to provide anything new. 

https://github.com/RedHatInsights/compliance-frontend/search?q=complianceemptystate&unscoped_q=complianceemptystate are the places where we use this and the client is not inherited from the state

Reply:
We could (and should at some point) also migrate to **not** use the `react-apollo` package.
The `@apollo/react-components` package provides a `Query` to use in the `SystemDetails` component.